manifests/role/webhost/fulcrum_www_and_app.pp

```diff
@@ -20,6 +20,15 @@
   include nebula::profile::www_lib::apache::base
   include nebula::profile::www_lib::apache::fulcrum
 
+  # Include a default vhost to catch monitoring requests by IP/fqdn.
+  # This is here rather than in the profile because it would be duplicate
+  # on www_lib_vm.
+  class { 'nebula::profile::www_lib::vhosts::default':
```

Comment:
It won't surprise you that I don't like parameters in roles, but the alternative would be creating new classes that map 1:1 to this role and to he www-lib role, and I think that path would be unreasonable and would gain us nothing of value. :shipit:

Reply:
Agreed, thanks. This can go away once fulcrum migrates out of www_lib_vm.